autoload/go/alternate.vim

```diff
@@ -0,0 +1,36 @@
+" By default use edit to switch to the new alternate buffer
```

Comment:
Can you change `edit` into `edit (current buffer view)` so people know what it means :)